### ISSUE TITLE
Application ignoring "Verify Certificate" option

### ISSUE THREAD

**Issue by pachulo** [theinvisible/openfortigui#38]
I've configured a connection and it works ok, but I wanted to verify the certificate from the server, so I ticked on the "Verify Certificate" checkbox of the "Certificate" section of VPN-Preferences and added some random stuff to the "Trusted-Cert (SHA256)"

![image](https://user-images.githubusercontent.com/3256953/40581285-6c84793c-6154-11e8-9307-212471a9daab.png)

The thing is that the connection works nevertheless...shouldn't it be failing if the checksum of the server certificate is not correct?

**Comment by theinvisible** [comment]
Hi,

this is a bug. In the current implementation the trusted cert is only used when user cert/key are provided. This should be fixed in next git commit and program release.

Cheers Rene

**Comment by SkyCrawl** [comment]
@theinvisible 

I can't find any documentation anywhere so, can I have two questions in regard to this?

1. Which certificate format is expected by the "CA-File" field above? I tried with DER but OpenFortiGUI wouldn't connect me (if I leave the field blank and untick "Verify Certificate", it will).
2. Which certificate format should we use as input for the checksum specified in the "Trusted-Cert" field? Again, I tried with DER but no luck.

Many thanks,
Jiri
